### ISSUE TITLE
Accessibility - unable to navigate and shift focus on to a specific page level element

### ISSUE THREAD

**Issue by ryanlynch717** [adaptlearning/adapt_framework#1752]
#1726 - Backbone.history.navigate and Adapt.trigger('router:navigateTo') does not allow us to scroll to and set foucs on any page level element; article, block or component. 

If that is the intention, the call to focusInitial should probably take an optional parameter of the selector so that a11y_focus can be called on it. See the call to focusInitial [here](https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/accessibility.js#L254)

The Adapt.scrollTo function [here] does something similar.
(https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/adapt.js#L128-L129)

**Comment by danielstorey** [comment]
Does Adapt.navigateToElement accomplish the above as it automatically calls .scrollTo after navigation

**Comment by oliverfoster** [comment]
is this an issue that still needs solving?

**Comment by oliverfoster** [comment]
note: ``Adapt.trigger('router:navigateTo') ``is now marked as deprecated
